.circleci/config.yml

```diff
@@ -173,6 +173,7 @@ jobs:
     environment:
       CONFIG_ENV: release
       PLAYWRIGHT_SKIP_DOWNLOAD: 'true'
+      SKIP_TSC: 'true'
```

Comment:
This was used on non-desktop builds in the past and my assumption is that this is now a requirement post https://github.com/Automattic/wp-calypso/pull/105476.